shared/src/commonTest/kotlin/InstrumentedTestsSuite.kt

```diff
@@ -255,12 +248,7 @@ class InstrumentedTestsSuite : FunSpec({
                     onNodeWithText(getString(Res.string.button_label_details)).performClick()
 
 
-                    val client = HttpClient {
-                        expectSuccess = true
-                        install(ContentNegotiation) {
-                            json()
-                        }
-                    }
+                    val client = testHttpClient
```

Comment:
Please inline variable, so that `client` isn't needed any more, but `testHttpClient` directly